Testing/VelaptorTests/Content/Caching/SoundCacheTests.cs

```diff
@@ -347,35 +357,37 @@ public void ShutDownNotification_WhenReceived_DisposesOfSounds()
             this.mockFile.Setup(m => m.Exists(soundPathA)).Returns(true);
             this.mockFile.Setup(m => m.Exists(soundPathB)).Returns(true);
 
-            this.mockShutDownReactable.Setup(m => m.Subscribe(It.IsAny<IReactor<ShutDownData>>()))
-                .Returns(this.mockShutDownUnsubscriber.Object)
-                .Callback<IReactor<ShutDownData>>(reactor =>
-                {
-                    if (reactor is null)
-                    {
-                        Assert.True(false, "Shutdown reactable subscription failed.  Reactor is null.");
-                    }
-
-                    shutDownReactor = reactor;
-                });
-
             var cache = CreateCache();
 
             cache.GetItem(soundPathA);
             cache.GetItem(soundPathB);
 
             // Act
-            shutDownReactor?.OnNext(default);
-            shutDownReactor?.OnNext(default);
+            this.shutDownReactor?.OnNext(default);
+            this.shutDownReactor?.OnNext(default);
 
             // Assert
             this.mockDisposeSoundReactable
                 .Verify(m =>
-                    m.PushNotification(new DisposeSoundData(11u), false), Times.Once);
+                    m.PushNotification(new DisposeSoundData(11u)), Times.Once);
             this.mockDisposeSoundReactable
                 .Verify(m =>
-                    m.PushNotification(new DisposeSoundData(22u), false), Times.Once);
-            this.mockShutDownUnsubscriber.Verify(m => m.Dispose(), Times.Once);
+                    m.PushNotification(new DisposeSoundData(22u)), Times.Once);
+        }
+        #endregion
+
+        #region Indirect Internal Tests
+        [Fact]
+        public void ShutDownReactable_WhenNotificationsEnded_DisposesOfUnsubscriber()
```

Comment:
Refactor unit test name to below:

```suggestion
        public void ShutDownReactable_WhenNotificationsEnd_DisposesOfUnsubscriber()
```